examples/ios/swift/AppleAuthentication/ContentView.swift

```diff
@@ -20,7 +20,7 @@ import SwiftUI
 import AuthenticationServices
 import RealmSwift
 
-/// Your MongoDB Realm App ID
+/// Your Atlas App Services app ID
```

Comment:
Is it not 'Atlas Device Sync'?

Reply:
this is the place for both...
/// Your Atlas Device Sync app ID
/// You can find it in Atlas App Services